example/descriptive_pipeline/server/main.py

```diff
@@ -258,10 +279,12 @@ async def websocket_endpoint(name: str, websocket: WebSocket):
                     result = {}
                     for index, (name, t) in enumerate(header):
                         if t.startswith('date'):
-                            result[name] = str(m[index]) # convert datetime type to string
+                            result[name] = str(
```

Comment:
let's put the comment in a separate line, and have  result[name] = str(m[index])  in one line.